google-cloud-compute/src/test/java/com/google/cloud/compute/v1/integration/ITSmokeInstancesTest.java

```diff
@@ -90,6 +96,85 @@ public void testInsertInstance() {
     assertInstanceDetails(resultInstance);
   }
 
+  @Test
+  public void testUpdateInstanceDescToEmpty() {
+    Instance resultInstance = insertInstance();
+    Assert.assertEquals("test", resultInstance.getDescription());
+    Assert.assertEquals(0, resultInstance.getScheduling().getMinNodeCpus());
+    Instance descInstance = resultInstance.toBuilder().setDescription("").build();
+    Operation updateOp =
+        instancesClient.update(DEFAULT_PROJECT, DEFAULT_ZONE, INSTANCE, descInstance);
+    waitUntilStatusChangeTo(updateOp);
+    Instance updated = getInstance();
+    assertInstanceDetails(updated);
+    Assert.assertEquals("", updated.getDescription());
+    Assert.assertEquals(0, resultInstance.getScheduling().getMinNodeCpus());
+  }
+
+  @Test
+  public void testResizeGroupToZero() throws IOException {
```

Comment:
Can you please put comments for tests like this (where it is not clear which specific issue they are trying to test), giving a brief explanation which type of potential failure they are testing?

Reply:
Done